gen/dcos-config-win.yaml

```diff
@@ -7,3 +7,209 @@ package:
+        ##   ie, when interval = "10s", precision will be "1s"
+        ##       when interval = "250ms", precision will be "1ms"
+        ## Precision will NOT be used for service inputs. It is up to each individual
+        ## service input to set the timestamp at the appropriate precision.
+        ## Valid time units are "ns", "us", "ms", "s".
+        precision = ""
+        ## Logging configuration:
+        ## Run telegraf with debug log messages.
+        debug = false
+        ## Run telegraf in quiet mode (error log messages only).
+        quiet = false
+        ## Specify the log file name. The empty string means to log to stderr.
+        logfile = ""
+        ## Override default hostname, if empty use os.Hostname()
+        hostname = ''
+        ## If set to true, do no set the "host" tag in the telegraf agent.
+        omit_hostname = false
+      # Read metrics about cpu usage
+      [[inputs.cpu]]
+        ## Whether to report per-cpu stats or not
+        percpu = false
+        ## Whether to report total system cpu stats or not
+        totalcpu = true
+        ## If true, collect raw CPU time metrics.
+        collect_cpu_time = false
+        ## If true, compute and report the sum of all non-idle CPU states.
+        report_active = false
+      # Read metrics about memory usage
+      [[inputs.mem]]
+      # Read metrics about disk usage by mount point
+      [[inputs.disk]]
+        ## Ignore mount points by filesystem type.
+        # TODO(branden): Do we want to ignore any fs types?
+        #ignore_fs = ["tmpfs", "devtmpfs", "devfs"]
+      # Read metrics about swap memory usage
+      [[inputs.swap]]
+      # Gather metrics about network interfaces
+      [[inputs.net]]
+      # Get the number of processes and group them by status
+      [[inputs.win_perf_counters]]
+      # Read metrics about system load & uptime
+      [[inputs.system]]
+      # Collect statistics about itself
+      [[inputs.internal]]
+        ## If true, collect telegraf memory stats.
+        collect_memstats = true
+
+           [[inputs.win_perf_counters.object]]
+              ObjectName = "Processor"
+              Instances = ["*"]
+              Counters = [
+                "% Idle Time",
+                "% Interrupt Time",
+                "% Privileged Time",
+                "% User Time",
+                "% Processor Time",
+                "% DPC Time",
+              ]
+              Measurement = "win_cpu"
+              IncludeTotal=true
+
+           [[inputs.win_perf_counters.object]]
+              ObjectName = "Diagnostics"
+              Instances = ["*"]
+              Counters = [
+                "Elapsed Time",
+                "% Processor Time",
+              ]
+              Measurement = "dcos_diagnostics"
+
+
+      [[inputs.docker]]
+         endpoint = "ENV"
+      
+      [[inputs.mesos]]
+      
+        timeout = 100
+         slaves = ["http://{{ local_ip }}:5051"]
```

Comment:
This is not provided as a variable currently. Where will this come from?

Reply:
This is copied from telegraf wich used on winagent by Oleksandr